### PR TITLE
Webconsole Content for Pipelines TP

### PR DESCRIPTION
**Pull request by nkakkar81**
This PR includes the Web Console content which will be added to the Pipelines doc for TP.

The PR is reviewed and approved by the QE, and has also been peer reviewed internally.

### CODE REVIEW

File: openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
Comment (ppitonak) [inline openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc:2]: missing `perspective` at the end

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (ppitonak) [inline modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:22]: IMHO there is inconsistent style comparing to the other docs. While the first `Pipeline Runs` is correct because this string is in dev console, `pipeline run` at the end of sentence should be IMHO written as `PipelineRun`. WDYT? The same applies to other kinds.

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (ppitonak) [inline modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:26]: The `Start Pipeline` dialog is displayed *only* if some input is needed. Maybe you could write it differently so that it's clear that in some situations it won't be displayed.

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (ppitonak) [inline modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:27]: Did you follow some `+Add` flow? This field name is quite specific and might be confusing for customers. We should decide whether it makes sense to keep this doc as general as possible or want to introduce some specific example.

File: openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
Comment (ppitonak) [inline openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc:18]: Just a reminder to update the link

**Review by ppitonak — CHANGES_REQUESTED** [review]
Please update `op-pipeline-list.png` so that it shows only the console, not the OS task bar, your bookmarks, URL of the cluster etc (or clean it up before taking the screenshot)

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (sanketpathak) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:27]: If you are taking reference from the +Add flow then this would be `APP_NAME` and mostly pre-populated

**Comment by sanketpathak** [comment]
Are we not targeting create flow in this pr

**Comment by nkakkar81** [comment]
> Are we not targeting create flow in this pr

@Preeticp  - Could you please reply to this query by sanket. Thanks

**Comment by Preeticp** [comment]
> Are we not targeting create flow in this pr

@sanketpathak as per our discussions with Andrew, on the tmp-odc-pipelines channel, Create flow is still DP for DevConsole, hence it was agreed that only this part of the dev console docs would go in.

**Comment by Preeticp** [comment]
> Please update `op-pipeline-list.png` so that it shows only the console, not the OS task bar, your bookmarks, URL of the cluster etc (or clean it up before taking the screenshot)

@ppitonak good catch, not sure how I missed that :/, we will crop out those details.

File: _topic_map.yml
Comment (Preeticp) [inline _topic_map.yml:736]: We can drop the articles in file names. But if you decide to change it, please ensure that all the ID and context values are changed too in the working-with-pipelines-using-the-developer-perspective.adoc

File: openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
Comment (Preeticp) [inline openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc:12]: Please comment this note out, this is part of the templates/create section in ODC which is still DP. Once we are TP, we will need to check and see if this limitation still exists.

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (Preeticp) [inline modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:8]: As Pavol mentions below, let's use uniform styles for PipelineRun as the rest of the Pipelines docs

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (Preeticp) [inline modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:11]: === /.

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (Preeticp) [inline modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:16]: Not your change, s/to/at the bottom

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (Preeticp) [inline modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:22]: Again, not your changes, but please make the following changes:
s/(icon image)/{kebab} In OCP docs repo this will render into the kebab icon.
Remove colon after to

Comment (Preeticp) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:22]: Agree, lets keep the styles for Pipelines, PipelineRun etc. consistent with the way we have changed it for the rest of the docs.

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (Preeticp) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:27]: @sanketpathak, Initially, these docs were created using Yaml files, then once add flow was in, instructions specific to that was added. See my commented out note on line 29, post our discussions. Now, the Add flow is in DP so it will not be documented. So we will need to keep only the documentation that is relevant for pipelines created using YAML or CLI.
@ppitonak for 4.4 we will need to keep this doc very general, once the add flow is TP we can add a specific example. Considering this, what do you suggest?

File: openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
Comment (Preeticp) [inline openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc:8]: {product-title}

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (Preeticp) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:26]: When I tested this, I saw the dialog box. I will test it again.

Comment (Preeticp) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:26]: @ppitonak on 4.4 cluster, I still see the Start modal. Can you please let us know in which situations it does not show?

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (nkakkar81) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:22]: I agree, will change this to be consistent with the rest of the doc.

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (nkakkar81) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:8]: yes..doing it for PipelineRun, Tasks etc

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (nkakkar81) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:11]: Dont you think it hsoud be the same level as the next topic? Maybe replace === with ==? wdty

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (nkakkar81) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:26]: @Preeticp - Can you pls confirm if this is valid?

File: _topic_map.yml
Comment (nkakkar81) [reply _topic_map.yml:736]: changed the IDs

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (Preeticp) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:27]: @nkakkar81 please comment out this section from line 26 onward. This is part of the Add flow which is DP for now.

File: openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
Comment (nkakkar81) [reply openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc:20]: I have commented out this content for now. I will add it back when we have the Installing Pipeline procedure merged.

**Comment by nkakkar81** [comment]
@ppitonak @sanketpathak @Preeticp - I have incorporated all your comments. Request you to please review and if you are ok with the changes, please approve ( /lgtm) . Thank you!

**Review by ppitonak — CHANGES_REQUESTED** [review]

File: openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
Comment (Preeticp) [inline openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc:2]: perspective

File: openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
Comment (Preeticp) [reply openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc:8]: web console

Comment (nkakkar81) [reply openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc]: done

Comment (nkakkar81) [reply openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc]: done

**Comment by nkakkar81** [comment]
Hi @ppitonak - I have incorporated your comments. Request you to please approve (/lgtm) if you do not have any additional comments. Thanks :)

**Review by ppitonak — APPROVED** [review]
/lgtm

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (bobfuru) [inline modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:16]: s/the bottom-right of the page/lower right of the page/
**lower left, lower right** _n_
Use “lower left” or “lower right,” not “bottom left” or “bottom right,” to refer to the location of an item in an
interface.

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (bobfuru) [inline modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:14]: The image text is small and hard to read, even on a large monitor. I haven't worked with images yet, so I'm not sure what our usage recommendations are. But I can't see the drop-down list at all. I suggest cropping the image to the area you are trying to show.

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (bobfuru) [inline modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:16]: s/It/This/

Comment (bobfuru) [inline modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:16]: s/to view the details of the Task/to view task details/

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (bobfuru) [inline modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:22]: s/Click *Pipeline/Click the *Pipeline/

Comment (bobfuru) [inline modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:22]: For readability, because the sentence is long, add "to" at the beginning of each phrase:
s/You can use the Options menu {kebab} to stop a running Pipeline, rerun a Pipeline using the same parameters and resources as that of the previous Pipeline execution, or delete a PipelineRun./You can use the Options menu {kebab} to stop a running Pipeline, to rerun a Pipeline using the same parameters and resources as that of the previous Pipeline execution, or to delete a PipelineRun.

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (bobfuru) [inline modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:24]: s/Click */Click the */

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (bobfuru) [inline modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:29]: s/drop-downs/drop-down lists/
"retain their default values" -- who is "their"? I'd suggest "the"

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (bobfuru) [inline modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:36]: s/as per/according to/
**as per** _prep_
Do not use. Use a phrase such as “according to,” “as,” or “as in.”

**Review by bobfuru — COMMENTED** [review]
Some convention callouts and questions about capitalization, along with a few other suggestions. Please let us review again after changes. Thanks!

File: _topic_map.yml
Comment (nkakkar81) [reply _topic_map.yml:748]: OpenShift Pipelines is a new Product which will go TP in 2-3 weeks :), that is why you did not find it in OCP docs. Its a name decided by the PMs/Stakeholders :)

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (nkakkar81) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:8]: Hi Bob, as we discussed in Slack, Openshift Pipelines is  a new product and does not refer to data pipeline.

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (nkakkar81) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:16]: Tasks again is a conceptual term for OpenShift Pipelines.

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (nkakkar81) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:14]: As discussed on slack, I will be replacing the images ( have already commented it out here ).

File: _topic_map.yml
Comment (nkakkar81) [reply _topic_map.yml:748]: Also, I will be commenting out the Pipelines section in topics.yaml before merging.

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (nkakkar81) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc]: done

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (nkakkar81) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc]: done

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (nkakkar81) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc]: done

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (nkakkar81) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc]: done

Comment (nkakkar81) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc]: done

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (nkakkar81) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc]: done

Comment (nkakkar81) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc]: done

Comment (nkakkar81) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc]: done

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc
Comment (nkakkar81) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc]: done

**Comment by nkakkar81** [comment]
Hi @bobfuru  -  As discussed over Slack, I have shared with you the conceptual module which explains the various keywords used in OpenShift Pipelines product docs. I have also incorporated most of the comments, and have provided explanation for the rest. Please let me know if you have any more comments/questions. Thanks

**Comment by bobfuru** [comment]
> Hi @bobfuru - As discussed over Slack, I have shared with you the conceptual module which explains the various keywords used in OpenShift Pipelines product docs. I have also incorporated most of the comments, and have provided explanation for the rest. Please let me know if you have any more comments/questions. Thanks

Hey @nkakkar81 - Thanks for the clarifications. Makes more sense now. :) 

Please squash commits and then let me know if you want this merged and cherrypicked to 4.4.

**Comment by nkakkar81** [comment]
Thank you @bobfuru  - I have commented out the topics.yaml file, and squashed the comments. Request you to please merge it, and you are right this is cherry picked to 4.4 :)

Thanks again!!!

**Comment by bobfuru** [comment]
LGTM

**Pull request merged**
